src/rtl/tbrowse.prg

```diff
@@ -123,6 +123,7 @@ CREATE CLASS TBrowse
    VAR cHeadSep   AS CHARACTER INIT ""          // 07. Heading separator characters
    VAR cColSep    AS CHARACTER INIT " "         // 08. Column separator characters
    VAR cFootSep   AS CHARACTER INIT ""          // 09. Footing separator characters
+   VAR cSepColor  AS String
```

Comment:
My Clipper skills may be a bit rusty, but:
1) By inserting an instance variable between 9 and 10 you break strict Clipper compatibility.
2) Not sure if AS STRING is any different from AS CHARACTER, probably it's better to keep the existing coding style
3) Your pull request will look much better if you squash multiple commits into one and/or resubmit it from a separate Git branch

Reply:
1. I put it on sep variables, I need add comment to show it is 13.
2. My mistake, when updating 3.2 code, it is CHARACTER
3. I do not know how group pull request as a single one, I was thinking this could be done easy by merge author.
It not possible group all as a single commit, I will create a new fork to do this.

Reply:
1. To keep low-level Clipper compatibility, addeng a comment won't help. I suspect that you need to move this instance variable below the line 141:
`/* === End of CA-Cl*pper compatible TBrowse instance area === */`
but maybe even below `#ifdef HB_COMPAT_C53`, line 162.
3. On git commit squashing: https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/

Reply:
Thanks. Can't use squash on this change, but will use it on fucture.